internal/libpng/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+include(FetchContent)
+
+FIND_PACKAGE(ZLIB)
```

Comment:
This should be required so that if it fails CMake does not continue generation (in case something went wrong source/Conan-wise and zlib couldn't be found).

Should also require a version as minimum, but I'm not sure if it's 100% necessary to version lock like that atm.